appdev/platform/index.rst

```diff
@@ -3,6 +3,8 @@
 Platform
 ========
 
+Here are Ubuntu Touch platforms key topics when you want to extend your app with Ubuntu Touch eco-system:
```

Comment:
I think ecosystem is just one word (at least that's what a quick search tells me)